mk/update-travis-yml.py

```diff
@@ -179,13 +185,25 @@ def get_linux_packages_to_install(compiler, arch):
             packages += [compiler + "-multilib",
                          replace_cc_with_cxx("linux", compiler) + "-multilib",
                          "linux-libc-dev:i386"]
+            if kcov == True:
```

Comment:
I think that this should be done outside the `if compiler.startswith("gcc-")` unless there's no hope of kcov ever working on a compiler other than gcc.